lib/players/computer.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+
+class Computer < Player
+
+	WIN_COMBINATIONS = [[0,1,2],
+      [3,4,5],
+      [6,7,8],
+      [0,3,6],
+      [1,4,7],
+      [2,5,8],
+      [0,4,8],
+      [6,4,2]]
+
+	def move (board)
+		
+		
+		valid_move = (1..9).select {|num| board.valid_move?(num)}
+
+			#binding.pry
```

Comment:
It's good to get in the habit of removing these comments before submitting a pull request.